incubation_process.md

```diff
@@ -18,16 +16,19 @@ Projects which intend to join the OpenRail Association are considered candidates
 
 ## Stage 1: Sandbox
 
-The Sandbox stage is the initial stage for projects being hosted by the OpenRail Association. Sandbox projects meet the minimal requirements necessary for being considered to be part of the OpenRail Association. The Sandbox stage is meant for projects that are in an early phase and are being prepared for meeting the full criteria of the Incubation stage.
+The Sandbox stage is the initial stage for projects being hosted by the OpenRail Association. Sandbox projects meet the minimal requirements necessary for being considered to be part of the OpenRail Association.
+
+The Sandbox stage is the entry stage for all projects hosted by the OpenRail Association. This stage is meant for projects to mature and work on meeting the criteria for the Incubation stage while already being part of the OpenRail Association. This can be projects that are in an early phase or it can be projects which already are mature. More mature projects will likely spend less time in the Sandbox stage.
 
 The barrier for entering the Sandbox stage is intended to be low, so that it's easy for projects to become part of the OpenRail Association and the work necessary for moving to the next stage can be done within the Association.
 
 ### Criteria
 
-* Describe project
-* Name maintainers of the project
-* Adopt the OpenRail Association Code of Conduct
-* Adhere to OpenRail Association IP Policy
+* **The project contributes to solving challenges of the railway sector in an open collaborative way so that it is in the scope of the purpose of the OpenRail Association.**
+* **The project is clearly described so that the OpenRail Association and potential users and contributors have a clear idea what it does.** The description and additional information is documented as part of the new project questionnaire.
+* **The project has named its maintainers so that it is clear who is taking decisions in the project and can add additional contributors.** The maintainers are documented in the project repository.
```

Comment:
Not "taking decisions" but "making decisions"

Reply:
ChatGPT suggests: The project has designated its maintainers to clarify who is responsible for making decisions and adding additional contributors. -> But this is not a style competition :-)

Reply:
Updated.